pandas/tests/io/formats/style/test_style.py

```diff
@@ -532,15 +532,17 @@ def test_set_na_rep(self):
         # GH 21527 28358
         df = DataFrame([[None, None], [1.1, 1.2]], columns=["A", "B"])
 
-        ctx = df.style.set_na_rep("NA")._translate()
+        with pytest.deprecated_call():
```

Comment:
can you use assert_produces_warning(FutureWarning)

Reply:
done